libs/common/src/sql-db-entities/fee-record-correction-request-transient-form-data/fee-record-correction-request-transient-form-data.types.ts

```diff
@@ -0,0 +1,9 @@
+import { RecordCorrectionRequestTransientFormData } from '../../types';
+import { DbRequestSource } from '../helpers';
+
+export type CreateFeeRecordCorrectionRequestTransientFormDataParams = {
```

Comment:
could this name not be shorter? It's good to be descriptive but this is pretty long, even to use in a file